src/org/usfirst/frc4904/robot/subsytems/PistonIntake.java

```diff
@@ -0,0 +1,27 @@
+package org.usfirst.frc4904.robot.subsytems;
+
+import org.usfirst.frc4904.robot.commands.PistonIntakeTransport;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class PistonIntake extends Subsystem {
+	public final Motor roller;
+	public final DoubleSolenoid piston;
+	public static final DoubleSolenoid.Value RELEASED = DoubleSolenoid.Value.kReverse;
+	public static final DoubleSolenoid.Value GRABBED = DoubleSolenoid.Value.kForward;
+	public static final double INTAKE_SPEED = -0.55;
+	public static final double OUTTAKE_SPEED = 0.45;
```

Comment:
how did you determine these values?

Reply:
We stole them from FloorIO. Is there something that they should be set to while we don't know the actual speeds?